packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1977,6 +1985,59 @@ export class ContainerRuntime
 		this.removeAllListeners();
 	}
 
+	/**
+	 * {@inheritDoc @fluidframework/runtime-definitions#IContainerRuntimeBase.isSnapshotFetchRequired}
+	 */
+	public isSnapshotFetchRequired(pathParts: string[]): boolean {
```

Comment:
Can we not have that API here?
See https://github.com/microsoft/FluidFramework/pull/20507/files#r1580262261 discussion on that.

Reply:
Removed from container runtime.